src/form/Form.js

```diff
@@ -196,6 +215,49 @@
         this.btntd
             .attr("colspan", this._maxCols * 2)
         ;
+        
+        // Disable Submit unless there is data
+        if (!this.allowEmptyRequest()) {
+            this.inputs().forEach(function(input, idx) {
```

Comment:
it needs to be applied to each input (its not a form.change event its a change on each element)